### ISSUE TITLE
Need to set default timezone

### ISSUE THREAD

**Issue by rolies106** [CouscousPHP/Couscous#119]
I get this error when run preview command : 

```
Warning: date(): It is not safe to rely on the system's timezone settings. You are *required* to use the date.timezone setting or the date_default_timezone_set() function. In case you used any of those methods and you are still getting this warning, you most likely misspelled the timezone identifier. We selected the timezone 'UTC' for now, but please set date.timezone to select your timezone. in phar:///couscous.phar/src/Model/WatchList/FinderWatch.php on line 21
```

Looks like you need to set default timezone at your script if machine is not define default timezone.

**Comment by mnapoli** [comment]
Thanks, we could indeed do something like this:

```php
if (! ini_get('date.timezone')) {
    date_default_timezone_set('UTC');
}
```

**Comment by rolies106** [comment]
Yeap, that would be great, but for quick fix I already edit my php.ini and set `UTC` as default timezone :)

**Comment by cyberbit** [comment]
:+1: @rolies106's solution worked for me, as well.

**Comment by mnapoli** [comment]
Was fixed by #130

**Comment by mnapoli** [comment]
This has been fixed in [1.3.2](https://github.com/CouscousPHP/Couscous/releases/tag/1.3.2).
